Makefile

```diff
@@ -13,14 +13,12 @@ ifndef BUILD_PATH
     export BUILD_PATH
 endif
 
-# Node command.
-NODE_COMMAND=--config=${CONFIG_PATH}
+NODE_COMMAND_ARGS=--config=${CONFIG_PATH}
 ifneq ($(SHARE_CONFIG),)
-  NODE_COMMAND+= --share-config=${SHARE_CONFIG}
+  NODE_COMMAND_ARGS+= --share-config=${SHARE_CONFIG}
 endif
 
-# Bootnode command.
-BOOTNODE_COMMAND=--config=${CONFIG_PATH}
+BOOTNODE_COMMAND_ARGS_ARGS=--config=${CONFIG_PATH}
```

Comment:
The variable name is a bit hard to read IMO

Reply:
Whoa, that's just a typo, nice that you spotted it!